dotnetv3/Lambda/README.md

```diff
@@ -0,0 +1,48 @@
+# AWS Lambda code examples for .NET
```

Comment:
# Lambda code examples for the SDK for .NET

Reply:
Fixed.